clients/callback_client.go

```diff
@@ -13,6 +13,7 @@ import (
 
 type CallbackClient struct {
 	httpClient *http.Client
+	Errors     chan error
```

Comment:
Since network operation is moved to separate goroutine i wanted to keep the original functionality. This channel serves to store result of async operation. do you think we should remove it until it is used?

Reply:
I think we can remove it and just log if the callback fails - there shouldn't be any other reason we need to know about callback errors